src/routeGeneration/templateHelpers.ts

```diff
@@ -237,7 +237,7 @@ export class ValidationService {
     const enumValue = members.find(member => {
       return member === String(value);
     });
-    if (!enumValue) {
+    if (enumValue === undefined) {
```

Comment:
shouldn't we also be checking for `null` too?

Reply:
I don't think so. enumValue is the result of a .find() it's either undefined, either a members, and I thing I should accept any value if it's a member.